app/views/brexit_checker/results.html.erb

```diff
@@ -76,10 +76,7 @@
 
     <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible govuk-section-break--bold">
     <div class="govuk-grid-row">
-      <div class="govuk-grid-column-one-third">
-        &nbsp; <!-- without this space, it will push the two-thirds column to the left -->
-      </div>
-      <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-grid-column-full">
```

Comment:
I don't think we need this, I think we can actually delete the surrounding `govuk-grid-column-full` and `govuk-grid-row`